plugins.md

```diff
@@ -63,6 +63,7 @@ Name | Description | Stars
 [sniff](https://github.com/eldadru/ksniff) | Start a remote packet capture on pods using tcpdump and wireshark | ![GitHub stars](https://img.shields.io/github/stars/eldadru/ksniff.svg?label=stars&logo=github)
 [sort-manifests](https://github.com/superbrothers/ksort) | Sort manifest files in a proper order by Kind | ![GitHub stars](https://img.shields.io/github/stars/superbrothers/ksort.svg?label=stars&logo=github)
 [ssh-jump](https://github.com/yokawasa/kubectl-plugin-ssh-jump) | A kubectl plugin to SSH into Kubernetes nodes using a SSH jump host Pod | ![GitHub stars](https://img.shields.io/github/stars/yokawasa/kubectl-plugin-ssh-jump.svg?label=stars&logo=github)
+[ssm-secret](https://github.com/pr8kerl/kubectl-ssm-secret) | Easily import/export of kubernetes secrets to/from AWS SSM Parameter Store path  | ![GitHub stars](https://img.shields.io/github/stars/pr8kerl/kubectl-ssm-secret.svg?label=stars&logo=github)
```

Comment:
If you change the short description, better update this as well. The file is auto-generated so that this version would not survive for long :)

Reply:
lol - was trying to be extra helpful. :) Updated with short description fwiw.